distributed/client.py

```diff
@@ -3035,8 +3044,15 @@ def _get_computation_code(
                 or fr.f_code.co_name in ("<listcomp>", "<dictcomp>")
             ):
                 continue
+
+            kwargs = dict(
```

Comment:
This `kwargs` dict feels awkward, but I don't see an obvious way of restructuring the code to remove the need for it. Thoughts?